full/Dockerfile

```diff
@@ -102,6 +102,7 @@ ENV GOPATH=/workspace:$GOPATH \
     PATH=/workspace/bin:$PATH
 
 ### Java ###
+## place .gradle and .m2 in /workspace because (1) that's a fast volume, (2) it survives workspace-restarts and (3) it can be warmend-up by pre-builds.
```

Comment:
Nit: How is `.m2` placed in `/workspace`?

Also, typos:
```suggestion
# Place .gradle and .m2 in /workspace because (1) that's a fast volume, (2) it survives workspace-restarts and (3) it can be warmed-up by pre-builds.
```

Reply:
fixed.